pages/draft/2019-09/migration-notes.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Draft 07 to Draft 2019-09
+section: docs
+---
+
+
+### Introduction
+
+JSON Schema Draft 2019-09 introduces several foundational changes from Draft 7. The most notable shifts include the concept of **vocabularies**, a new **meta-schema model**, and the introduction of **recursive referencing** (`$recursiveRef`, `$recursiveAnchor`). These changes reflect greater modularity, formal extensibility, and clarity when validating recursive data structures like trees or graphs.
+
+This guide covers the key differences and provides step-by-step migration instructions to help you transition your schemas and tooling from Draft 7 to Draft 2019-09.
+
+
+### Keyword Changelog
+
+| Keyword (Draft 7)        | Keyword (Draft 2019-09)  | Specification | Keyword Type     | Behavior Details                                                                     |
+| ------------------------ | ------------------------ | ------------- | ---------------- | ------------------------------------------------------------------------------------ |
+| `$schema`                | `$schema`                | Core          | Identifier       | Updated to use 2019-09 meta-schema: `"https://json-schema.org/draft/2019-09/schema"` |
+| *not present*            | `$vocabulary`            | Core          | Identifier       | Declares which vocabularies a schema uses. Enables modular extensibility.            |
+| *not present*            | `$recursiveRef`          | Core          | Identifier       | Replaces `$ref` in recursive contexts.                                               |
+| *not present*            | `$recursiveAnchor`       | Core          | Identifier       | Used in base schemas to define recursive entry points.                               |
+| `definitions`            | `definitions`            | Validation    | Object Container | Still supported but now superseded by `$defs` in 2020-12+.                           |
```

Comment:
I don't think the spec mentions `definitions` at all. We should suggest `$defs`. Also `$defs` was introduced in 2019-09 and not 2020-12 as this page says